server/src/test/java/org/opensearch/index/IndexSettingsTests.java

```diff
@@ -777,9 +775,30 @@ public void testRemoteStoreExplicitSetting() {
         assertTrue(settings.isRemoteStoreEnabled());
     }
 
+    public void testRemoteTranslogStoreDefaultSetting() {
```

Comment:
Should we add a test if settings is null?

Reply:
@nknize added tests for null settings